src/components/Order_page/index.js

```diff
@@ -0,0 +1,28 @@
+import {useEffect, useState} from "react";
```

Comment:
Get rid of all semi colons to keep with consistency of the other files (we went through and deleted most of hem earlier)